content/issues/organizing-your-work-with-project-boards/managing-project-boards/changing-project-board-visibility.md

```diff
@@ -23,6 +23,12 @@ shortTitle: Change visibility
 
 {% endtip %}
 
+{% tip %}
```

Comment:
Hi @obtusedev! It's not the case that repository-based project boards are public by default. Instead, they have the same visibility as the repository they are associated with... if the repository is private, the project board will be private, too. If the repository is public, so is the project board.

Instead of adding this tip, let's update the reusable above, `{% data reusables.project-management.project-board-visibility %}`, to specify that the current content applies to project boards owned by organizations and user accounts only, while project boards associated with repositories will share the visibility of their repository, and that cannot be changed.

Reply:
Hi, sorry for the late reply. I didn't see the notification until now. I went ahead and made a [PR](https://github.com/github/docs/pull/10129) fixing the issues using your suggestions as a guideline.

Reply:
Great, that new PR will get added to the review board, and I'll close this one!